src/dbsync/tests/string_helper/string_helper_test.h

```diff
@@ -0,0 +1,12 @@
+#include "gtest/gtest.h"
```

Comment:
Add #pragma here or the: 
#ifndef _STRING_HELPER_TEST_H
#define _STRING_HELPER_TEST_H
...
#endif

Reply:
Good catch!